src/lib/numbers/__tests__/clamp.test.ts

```
@@ -0,0 +1,52 @@
+    expect(clamp(0, { min: 0, max: 10 })).toBe(0);
+    expect(clamp(10, { min: 0, max: 10 })).toBe(10);
+  });
+
+  it("works with non-integer values", () => {
+    expect(clamp(2.5, { min: 1.2, max: 3.4 })).toBeCloseTo(2.5);
+    expect(clamp(0.5, { min: 1.2, max: 3.4 })).toBeCloseTo(1.2);
+    expect(clamp(5.6, { min: 1.2, max: 3.4 })).toBeCloseTo(3.4);
+  });
+
+  it("handles Infinity and -Infinity correctly", () => {
+    expect(clamp(Infinity, { min: -100, max: 100 })).toBe(100);
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)